### DISCUSSION TITLE
Upgrade Seq version 2021.1 to 2021.2

### DISCUSSION THREAD

**Discussion by mpp-olano** [datalust/seq-tickets#1330] [Help]
Need to upgrade Seq Docker Image from 2021.1 to 2021.2 on Linux VM. What is the recommended steps to go about it? I have pulled the 2021.2 image from Docker Hub.

**Comment by nblumhardt** [comment]
Hi @mpp-olano, thanks for getting in touch! There are no special steps required - 2021.1 and 2021.2 are fully-compatible. It should just be a matter of starting up the new container version, pointing to the same persistent volume.

https://stackoverflow.com/questions/26734402/how-to-upgrade-docker-container-after-its-image-changed has all of the details regarding container upgrades, if you need some additional info on how to get Docker to do this.

Hope this helps,
Nick
